proto/cosmos/base/tendermint/v1beta1/query.proto

```diff
@@ -7,6 +7,7 @@ import "tendermint/p2p/types.proto";
 import "tendermint/types/block.proto";
 import "tendermint/types/types.proto";
 import "cosmos/base/query/v1beta1/pagination.proto";
+import "cosmos/tx/v1beta1/tx.proto";
```

Comment:
one question I have is that the tmservice is meant to be a simple proxy to TM, and not app-specific. I'm not still it should do tx decoding.

For example, the `/txs` endpoint to fetch txs, exposed by tendermint, is not in this tmservice, but in the Tx service. That's imo the service that decodes txs. I'm wondering if we shouldn't create a new GetBlockWithTxs there. (it's also where the `protoTxProvider` is currently used).

any thoughts @alexanderbez @technicallyty ?

Reply:
Yeah I agree here @amaurym. I don't think we should be adding custom business logic to TM proxy endpoints. I think that's a good point.

Reply:
yeah i was thinking this as well. maybe a solution could be to add a query string to the request for decoding, something like: `GET block/{height}.?decode=true`?

Reply:
pinging here - any thoughts on previous comment? @alexanderbez @amaurym

Reply:
No I don't think we should modify Tendermint proxy queries. This should be an entirely new query/route. I believe this is what @amaurym was suggesting?

Reply:
Yeah, same, I think `tx.Tx` should not appear in this query.proto file.